scam-detector-py/src/agent.py

```diff
@@ -569,7 +569,7 @@ def emit_passthrough_finding(w3, alert_event: forta_agent.alert_event.AlertEvent
                 logging.info(f"{BOT_VERSION}: alert {alert_event.alert_hash} {alert_event.bot_id} {alert_event.alert.alert_id} - url {scammer_url} already alerted on for {alert_id}; skipping")
                 continue
 
-            if Utils.is_fp(w3, scammer_url):
+            if Utils.is_fp(w3, scammer_url, CHAIN_ID, False):
```

Comment:
We're passing a url to the `.is_fp`, so I added a flag to the function to avoid checking Etherscan API, but only check the `Utils.is_in_fp_mitigation_list`